saleor/product/migrations/0096_product_minimal_variant_price.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.1 on 2019-06-10 11:23
+
+import django_prices.models
+from django.db import migrations
+
+
+def populate_product_minimal_variant_price(apps, schema_editor):
+    Product = apps.get_model("product", "Product")
+    for product in Product.objects.prefetch_related("variants").iterator():
```

Comment:
First of all. AFAIK `prefetch_related` and `iterator` will not work together. 
Secondly - Do you need to fetch `variants` here?